GitCommands/IDiffListSortService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitCommands
+{
+    public enum DiffListSortType
```

Comment:
We strive for one public class per file (at least for the new submissions).
Interfaces live in the same file as their implementations (unless there are multiple implementations).

Reply:
So to be clear you want the enum and extensions class in their own files and the interface moved to the class's file?

Reply:
Correct